probcal/data_modules/aaf_datamodule.py

```diff
@@ -0,0 +1,101 @@
+from pathlib import Path
+
+import lightning as L
+import numpy as np
+from torch.utils.data import DataLoader
+from torch.utils.data import Subset
+from torchvision.transforms import AutoAugment
+from torchvision.transforms import Compose
+from torchvision.transforms import Normalize
+from torchvision.transforms import Resize
+from torchvision.transforms import ToTensor
+
+from probcal.custom_datasets import AAFDataset
+from probcal.custom_datasets import ImageDatasetWrapper
+
+
+class AAFDataModule(L.LightningDataModule):
+
+    IMG_SIZE = 224
+
+    def __init__(
+        self,
+        csv_file_path: str | Path,
+        root_dir: str | Path,
+        batch_size: int,
+        num_workers: int,
+        persistent_workers: bool,
+        surface_image_path: bool = False,
+    ):
+        super().__init__()
+        self.csv_file_path = csv_file_path
+        self.root_dir = Path(root_dir)
+        self.batch_size = batch_size
+        self.num_workers = num_workers
+        self.persistent_workers = persistent_workers
+        self.surface_image_path = surface_image_path
+
+    def prepare_data(self) -> None:
+        # Force images to be downloaded.
```

Comment:
As far as I can tell, creating the `AAFDataset` doesn't force images to be downloaded, right?

Reply:
Fixed